.github/workflows/release-on-tag.yml

```
@@ -0,0 +1,119 @@
+      - name: Install Android SDK platform & build-tools
+        run: |
+          sdkmanager "platforms;android-36" "build-tools;36.0.0"
+
+      - name: Set up Gradle
+        uses: gradle/actions/setup-gradle@v4
+
+      # Optional: only present once the owner has generated the release key
+      # (RELEASE.md item 1) and added it as repo secrets. Absent, the build
+      # below falls back to an unsigned APK — same as CI, see build.gradle.kts.
+      - name: Decode release signing key
+        if: ${{ secrets.SKYWARD_RELEASE_STORE_FILE_BASE64 != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)